### PR TITLE
partner interventions budget

### PR DESCRIPTION
**Pull request by ntrncic**
finish unicef/etools-issues#32; finish unicef/etools-issues#26

### CODE REVIEW

**Comment by robertavram** [comment]
@ntrncic 
planned budget only for current year

File: EquiTrack/partners/validation/agreements.py [hunk 1]
```diff
@@ -80,6 +80,17 @@ def start_date_equals_max_signoff(agreement):
         return False
     return True
 
+def end_date_pca_validation(agreement):
+    if agreement.agreement_type == agreement.__class__.PCA and agreement.start:
+        cp = CountryProgramme.objects.filter(from_date__lte=agreement.start,
+                                             to_date__gte=agreement.start,
+                                             wbs__contains='/A0/').order_by('id').first()
+        if cp:
+            if cp is None or agreement.end is None or agreement.end > cp.to_date:
```
Comment (robertavram) [inline EquiTrack/partners/validation/agreements.py]: if statements contradict eachother here

File: EquiTrack/partners/views/v2.py [hunk 2]
```diff
@@ -210,8 +210,8 @@ def get(self, request):
                                                                                                    'email'))
         hrps = list(ResultStructure.objects.values())
         current_country_programme = CountryProgramme.current()
-        cp_outputs = list(Result.objects.filter(result_type__name=ResultType.OUTPUT, wbs__isnull=False,
-                                                country_programme=current_country_programme).values('id', 'name', 'wbs'))
+        cp_outputs = list(Result.objects.filter(result_type__name=ResultType.OUTPUT, wbs__isnull=False)
+                          .values('id', 'name', 'wbs', 'country_programme'))
```
Comment (robertavram) [inline EquiTrack/partners/views/v2.py]: why are we changing this?

Comment (ntrncic) [inline EquiTrack/partners/views/v2.py]: removed

**Pull request merged**